product_docs/docs/biganimal/release/using_cluster/managing_replicas.mdx

```diff
@@ -0,0 +1,137 @@
+       - While promoting a replica to a cluster, you can't modify the **Cloud Provider**, **Postgres Type**, **Postgres Version**, and **Region** fields.
+
+1. Select **Promote Replica**. 
+
+
+
+## Modify a faraway replica
+
+1. Sign in to the [BigAnimal portal](https://portal.biganimal.com/).
+
+1. Go to the [Clusters](https://portal.biganimal.com/clusters) page. A list of previously created clusters appears.
+
+1. Select the cluster with the replica you want to modify. In the **Overview** tab, you can see the cluster's replicas under **Faraway Replicas**.
+
+1. Select the **Edit replica** icon next to the replica you want to modify. The Edit Replica page appears. 
+
+1. In the **Replica Settings** tab, in the **Faraway Replica Name** field, enter the name for your replica.
+
+1. In the **Password** field, enter a password for your replica. 
+ 
+   In the **Database Type** section, you can see the Postgres type and Postgres version inherited from the parent cluster. You can't edit the Postgres type and Postgres version. 
+
+1. In the **Region** section, select the region where you want to deploy your replica.
+
+   You can select a region for deploying your replica even if it is not yet *Active*. Your cluster creation request is added to a queue, and the replica is created after you activate the region. See [Activating regions](/biganimal/latest/getting_started/activating_regions) for more information.
+ 
+1. The fields in the **Instance Type** section are populated with values inherited from the parent cluster. You can edit the fields in the **Instance Type** section. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on the instance type settings. 
+
+   !!! Note
+       To avoid lag during replication, ensure that your replicas' instance type is at least as large as the parent cluster's instance type.
+
+1. The fields in the **Storage** section are populated with values inherited from the parent cluster. You can edit the fields in the **Storage** section. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on the storage settings.
+
+   !!! Note
+       To avoid lag during replication, ensure that your replicas' storage type is at least as large as the parent cluster's storage type.
+ 
+1. In the **Networking** section, you specify whether to use private or public networking. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on the Networking settings.
+
+1. On the DB Configuration tab, to avoid replication issues when running a replica (standby server), set the following database configuration parameters to the same or higher value than on the parent cluster (primary server). Otherwise, queries are not allowed in the replica. 
```

Comment:
- could we use "source cluster" instead of "parent cluster"?
- what do you mean by "queries" are not allowed in the replica? Do you mean modifying any of the GUCs?
- in this section and the Create Replica section, could we add a note that if the user is going to increase these GUC values we highly recommend they set the same value for all faraway replicas and then set the value on the source cluster. And that if the user is going to decrease these GUC values, we recommend setting the source cluster's value first and then the faraway replicas

Reply:
> * could we use "source cluster" instead of "parent cluster"?
@amrita-suresh , do we want to use this terminology throughout the faraway replica sections? that would change 17 instances. Most of the instances are in the Managing replicas topic, but there is one on the Faraway replicas topic in the Overview section.
> * what do you mean by "queries" are not allowed in the replica? Do you mean modifying any of the GUCs?
@neel5481 can you help answer this question?

Reply:
! addressed the third bullet.